package-lock.json

```diff
@@ -1652,6 +1652,12 @@
       "integrity": "sha512-v1plID3y9r/lPhviJ1wrXpLeyUIGAZ2SHNYTEapm7/8A9nLPoyvVp3RK/EPFqn5kEznyWgYZNsRtYYIWbuG8KA==",
       "dev": true
     },
+    "cypress-axe": {
```

Comment:
This appears to be in the wrong package - I'm not against this here if it's correct, but just check it

Reply:
@shilpabasu22 can you confirm this?

Reply:
I have added this under lpa-questionnaire-e2e-tests and not under main package.json file. Not sure why is it showing in that package-lock.json file. I can remove it or touch base with you tomorrow.